src/tunnels/views.py

```diff
@@ -75,7 +69,8 @@ class ReverseServerAuthorizedKeysConfig(APIView):
     )
     def get(self, request, server_id, ssh_server_hostname):
         try:
-            server_auth_key = ReverseServerAuthorizedKeys.objects.get(id=server_id)
+            user = request.user
+            server_auth_key = ReverseServerAuthorizedKeys.objects.get(id=server_id, user=user)
```

Comment:
It seems a little redundant. Can it be extracted to a super-class's method?

Reply:
In this commit https://github.com/NatLee/telepy/pull/17/commits/eb8442c1f4dc319c23aec029242baaf44d802b63, I add `ReverseServerScriptBase` for script and config inheriting.

But there is one problem that the checking logic makes the code more complicated. 🥲